src/js/modules/articles/components/ArticleBody.js

```diff
@@ -93,42 +108,29 @@ const styles = {
   },
 };
 
-const ArticleBody = ({
-  classes,
-  article: { content, title },
-  articles,
-  media,
-}) => {
+const ArticleBody = ({ classes, data, article }) => {
   const isCarouselButtonVisible =
-    SPEC_IMG_CAROUSEL_PATTERN.test(content) && Object.values(media).length > 0;
-  const referencedArticleId = SPEC_REFERENCE_PATTERN.test(content)
-    ? parseInt(SPEC_REFERENCE_PATTERN.exec(content)[1])
-    : null;
-  let referencedArticle = null;
-  if (referencedArticleId) {
-    referencedArticle = articles.find(
-      article => article.id === referencedArticleId,
-    );
-  }
+    SPEC_IMG_CAROUSEL_PATTERN.test(article.content) && Object.values(media).length > 0;
```

Comment:
Yeah...you see how sorta finicky this is? Why I want a straightforward boolean in the model